src/docs/ftw-legacy/legacy-overview/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Legacy templates
+slug: legacy-templates
+updated: 2023-01-01
+category: ftw-legacy
+ingress:
+  This article describes the Flex Templates for web that are deprecated
+  on the release of the new Sharetribe Web Template.
+published: true
+---
+
+## Deprecated templates
+
+The Sharetribe Web Template replaces three separate templates that were
+previously used for daily bookings
+([ftw-daily](https://github.com/sharetribe/ftw-daily)), product
+purchases ([ftw-product](https://github.com/sharetribe/ftw-product)),
+and hourly bookings
+([ftw-hourly](https://github.com/sharetribe/ftw-hourly)). The new
+template combines the functionality of these three templates, allowing
+you to configure it for either product bookings or hourly and daily
+bookings by making simple changes to the configuration files.
+
+If you are just beginning to develop a marketplace, we recommend that
+you start with the Sharetribe Web Template. If you have already built
+your marketplace on one of the legacy templates, you can still continue
+use it with Sharetribe APIs and our support will continue to help you
+with development related issues. In addition, you will always be able to
```

Comment:
Would the "you will always be able to access the documentation" approach mean that we need to host the full documentation somewhere ourselves? I can't remember whether we decided to host it as a deploy preview branch (which might have an expiry date) or as a separate subfolder somewhere on the Docs site.

Reply:
I now put in a mention that documentation will be available until 12/2024, at which point we can decide to either deprecate the legacy documentation or continue to keep it available.